src/Simplex/Chat.hs

```diff
@@ -3753,7 +3771,7 @@ cancelSndFileTransfer user@User {userId} ft@SndFileTransfer {fileId, connId, age
           void . sendDirectMessage conn (BFileChunk sharedMsgId FileChunkCancel) $ ConnectionId connId
         _ -> withAgent $ \a -> void . sendMessage a acId SMP.noMsgFlags $ smpEncode FileChunkCancel
       pure fileConnId
-    fileConnId = if isJust fileInline then Nothing else Just acId
+    fileConnId = if isNothing fileInline then Just acId else Nothing
```

Comment:
not sure if this should check for xftp as well?

Reply:
and not sure why this change is needed at all

Reply:
that it's not xftp is checked in cancelSndFile, this one is per connection checking XFTP send here would require passing additional data to it